firefly_client/firefly_client.py

```diff
@@ -175,7 +113,7 @@ def _handle_event(self, ev):
     #    'dataType': ['STRING', 'JSON', 'BG_STATUS'],
     #    'data': {'channel': , 'connID': }
     # }
-    def received_message(self, m):
+    def _received_message(self, m):
```

Comment:
The prepending of the underscore is understandable as wanting to make this a non-public method. However it causes the parent class's received_message docstring to appear with a note that 'You should override this method in your subclass.'